x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -26,7 +26,7 @@ async function getApmAnomalyDetectionJobs(
     const { jobs } = await ml.anomalyDetectors.jobs(APM_ML_JOB_GROUP_NAME);
     return jobs;
   } catch (error) {
-    if (error.statusCode === 404) {
+    if (error.statusCode === 404 || error.statusCode === 403) {
```

Comment:
We should have an API test to guard this that runs as a read-only user.

Reply:
oh, this is for 7.8 - so an API test is probably overkill for this fix. But going forward we should have it.